<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite thread-count="1" guice-stage="DEVELOPMENT" name="Failed suite [SFDC Test Automation Test Suite]" parallel="methods">
  <listeners>
    <listener class-name="com.qa.EventListener.TestListener"/>
    <listener class-name="com.qa.EventListener.FailureRetryListener"/>
  </listeners>
  <test name="Log a Call Test Scenarios(failed)" parallel="methods">
    <classes>
      <class name="com.crm.qa.testcases.LogActivityTest">
        <methods>
          <include name="LogaCallwithSpouse_ValidateAtPrimary"/>
          <include name="CreateRetail1Account_Spouse"/>
          <include name="tearDown"/>
          <include name="OnlyCreateRetail_2_Account"/>
          <include name="UpdatePrimaryAccountdetails"/>
          <include name="reachedwithBusinessLead_ScheduleMeeting"/>
          <include name="setUp"/>
          <include name="UpdateSpouseAccountdetails"/>
          <include name="LogACallSpouse"/>
        </methods>
      </class> <!-- com.crm.qa.testcases.LogActivityTest -->
    </classes>
  </test> <!-- Log a Call Test Scenarios(failed) -->
</suite> <!-- Failed suite [SFDC Test Automation Test Suite] -->
